src/App.tsx

```diff
@@ -20,6 +20,7 @@ function App() {
         <main className="ml-64 grow overflow-y-scroll">
           <Routes>
             <Route path="/documents/*" element={<Documents />} />
+            <Route path="/create-project" element={<CreateProject />} />
```

Comment:
It would be nice if the URL is a little simpler. How about`/projects/new`.

- https://sentry.io/organizations/yorkie-team/projects/new/
- https://github.com/new